### PR TITLE
Endpoint to create a task

### PR DESCRIPTION
**Pull request by pbochis**

### CODE REVIEW

**Comment by flowlo** [comment]
Check for authorization missing. Any bloody spammer can fill our DB with spam this way.

**Comment by flowlo** [comment]
Also, model.CodeTask will go away.

File: main.go [hunk 1]
```diff
@@ -65,6 +65,7 @@ func main() {
 
 	r.HandleFunc("/tasks/{key}", setup(controllers.TaskByKey))
 	r.HandleFunc("/tasks", setup(controllers.Tasks))
+	r.HandleFunc("/task/code", setup(controllers.CreateCodeTask))
```
Comment (flowlo) [inline main.go]: This should be `/tasks/...` otherwise it's inconsistent with every other route.

File: controllers/codeTask.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		Flags     string
+		Languages []string
+		Runner    string
+	}{}
+
+	if err := json.NewDecoder(r.Body).Decode(&body); err != nil {
+		return http.StatusBadRequest, err
+	}
+
+	task := model.CodeTask{
```
Comment (flowlo) [inline controllers/codeTask.go]: `undefined: model.CodeTask`

File: main.go (hunk 1 above)
Comment (pbochis) [inline main.go]: I already changed most of this. Waiting for your PR to get on the master so i can rebase and push

Comment (flowlo) [inline main.go]: Then you should review my PR.

**Pull request closed without merging**